src/platforms/dotnet/migration.mdx

```diff
@@ -46,65 +44,246 @@ SentrySdk.Init(o =>
+- Captures unhandled exceptions in the middleware pipeline
+- Captures exceptions handled by the framework UseExceptionHandler and Error page display.
+- Any event sent will include relevant application log messages
+- RequestId as tag
+- URL as tag
+- Environment is automatically set (IHostingEnvironment)
+- Request payload can be captured if opt-in
+- Support for EventProcessors registered with DI
+- Support for ExceptionProcessors registered with DI
+- Captures logs from the request (using Microsoft.Extensions.Logging)
+- Supports configuration system (e.g: appsettings.json)
+- Server OS info sent
+- Server Runtime info sent
+- Request headers sent
+- Request body compressed
+
+All packages are:
+
+- Strong named
+- Tested on Windows, Linux and macOS
+- Tested on .NET Core, .NET Framework and Mono
+
+It's also very configurable, for example:
+
+- HTTP Proxy
+- Event sampling
+- Enable request body extraction
+- Send PII data (Personal Identifiable Information, requires opt-in)
+- Read diagnostics activity data
+- BeforeSend: Callback to modify/reject event before sending
+- BeforeBreadcrumb: Callback to modify/reject a breadcrumb
+- LogEventFilter: Filter events by inspecting log data
+- Maximum number of breadcrumbs to store
+- Event queue depth
+- Shutdown timeout: If there are events to send, how long to wait until shutdown
+- Accept compressed response
+- Compress request body
+- Breadcrumb level: Minimum log level to store as a breadcrumb
+- Event level: Minimum log level to send an event to Sentry
+- Disable duplicate event detection
+- Disable capture of global unhandled exceptions
+- Add event processor
+- Add exception processor
+- Enable SDK debug mode
+- Attach stack trace for captured messages (opt-in)
+
+### Breaking Changes
+
+Upgrading from SharpRaven to Sentry SDK includes the following breaking changes.
+
+#### .NET Framework Support Reduced
+
+If your project is targeting .NET Framework, it will need to be using at least v4.6.1 to use Sentry (SharpRaven supported .NET Framework v3.5 and higher).
+
+#### Initialization
+
+Changed how to initialize Sentry:
+
+```csharp
+using Sentry;
+
+// Initialize with DSN
+SentrySdk.Init("___PUBLIC_DSN___");
+
+// Initialize with options
+SentrySdk.Init(o =>
+{
+    o.Dsn = "___PUBLIC_DSN___";
+});
+```
+
+For ASP.NET (classic) integration, add `Sentry.AspNet` package and initialize it by calling `o.AddAspNet()`:
+
+```csharp
+using Sentry;
+using Sentry.AspNet;
+
+SentrySdk.Init(o =>
+{
+    o.Dsn = "___PUBLIC_DSN___";
+
+    o.AddAspNet();
+});
+```
+
+#### Event Queue
+
+Contrary to SharpRaven, events in Sentry SDK are sent in background without blocking the currently executing code. Calling `SentrySdk.CaptureException(...)` does not immediately send an event, but instead queues it up on a background thread.
+
+Usage changes:
+
+```csharp
+// Before
+ravenClient.Capture(...);
+await ravenClient.CaptureAsync(...);
+
+// After
+SentrySdk.CaptureEvent(...);
+```
+
```

Comment:
```suggestion

If you need to stop the execution until events captured are sent to Sentry, you can also flush the queue:

```csharp
SentrySdk.CaptureException(...);
// Only proceed once exception is captured. Waiting up to 2 seconds for it.
await SentrySdk.FlushAsync(TimeSpan.FromSeconds(2));
```